doc/video.md

```diff
@@ -29,24 +31,32 @@ sdk.on('sessionStarted', function (session) => {
     sessionEventsToLog.forEach((eventName) => {
       session.on(eventName, (e) => console.info(eventName, e));
     });
-    sdk.acceptSession({ id: session.id, audioElement, videoElement });
+    sdk.acceptSession({ sessionId: session.id, audioElement, videoElement });
   }
 });
+
 await sdk.startVideoConference(roomJid);
 ```
 
 At this point, you should be in the conference and able to see and hear others. Here are the various events
 and functions that are relevant to video conferences.
 
-#### Relevant Events
+## Video Session Level Events
+
+These are `session` level events and are in addition to the already documented [session level events].
 
-`session.on('participantsUpdate', (IParticipantsUpdate) => {})` - This event will happen whenever a conversation update event is received.
+#### `participantsUpdate`
+This event will happen whenever a conversation update event is received.
```

Comment:
Is this accurate? I thought this only happened if the number of participants changed or something on the participants changed. TBF I think that covers like 99% of the cases.